scripts/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"sort"
+
+	"github.com/jszwec/csvutil"
+)
+
+type Repository struct {
+	Repo     string `csv:"repo"`
+	Metadata string `csv:"metadata,omitempty"`
+}
+
+// Checks for duplicate item in the projects.txt
+// This is used in the builds to validate there aren't duplicates in projects.txt.
+func main() {
+	projects, err := os.OpenFile(os.Args[1], os.O_RDONLY, 0644)
+	if err != nil {
+		panic(err)
+	}
+	defer projects.Close()
+
+	repos := []Repository{}
+	data, err := ioutil.ReadAll(projects)
+	if err != nil {
+		panic(err)
+	}
+	err = csvutil.Unmarshal(data, &repos)
+	if err != nil {
+		panic(err)
+	}
+	sort.Slice(repos, func(i, j int) bool {
```

Comment:
We don't need to sort since we are using the map to determine duplicates. 

Alternatively, sort and then detect duplicates if there are 2 of the same next to each other (without using the map). 

Using the map alone seems much simpler to me though.

Reply:
Agreed